EIPS/eip-3788.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: 3788
+title: Strict enforcement of chainId
+description: Strictly enforce the chainId on transactions based on the chain.
```

Comment:
I think the abstract explains the goal better. I had no idea what this EIP does based on this line.